backend/spiritual_avatar_generation_engine.py

```
@@ -30,13 +33,22 @@ class SwamjiAvatarGenerationEngine:
     Real D-ID + ElevenLabs Integration
     """
     
-    def __init__(self):
+    def __init__(self, settings: AppSettings, db_session_manager: DatabaseManager):
```

Ruff (0.12.2)

Redefinition of unused `settings` from line 22

(F811)